modules/oc-mirror-imageset-config-parameters-v2.adoc

```diff
@@ -0,0 +1,396 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-imageset-config-parameters-v2_{context}"]
+= ImageSet configuration parameters for v2
+
+The oc-mirror plugin requires an image set configuration file that defines what images to mirror. The following table lists the available parameters for the `ImageSetConfiguration` resource.
+
+.`ImageSetConfiguration` parameters
+[cols="2,2a,1a",options="header"]
+|===
+|Parameter
+|Description
+|Values
+
+|`apiVersion`
+|The API version for the `ImageSetConfiguration` content.
+|String. For example: `mirror.openshift.io/v1alpha2`.
```

Comment:
since this is for v2, better to use v2 api version :  
`mirror.openshift.io/v2alpha1`

Reply:
I will change accordingly.